docs/admin/reference/backup.rst

```diff
@@ -24,7 +24,7 @@ Preserve files from ``dom0`` and ``sd-gpg``
 
 Preserve configuration files and private key material by copying them into ``dom0``.
 
-In a ``dom0`` Terminal via **Q ▸ Terminal Emulator**:
+In a ``dom0`` Terminal via **Q > Gear Icon (left-hand side) > Other Tools > Xfce Terminal**:
```

Comment:
The ▸ is arguably nicer, but it was only used in this chapter, so I've replaced it for easier search & replace across the docs. If folks prefer, I can do a global search & replace to use ▸ instead (it may be time to start a style guide ..).

Reply:
I agree the ▸ is preferable to the > symbol, although I'd be curious if any others on the team have strong opinions one way or the other. For now, I agree that consistency is the most important - thanks for catching.

And a big +1 to a style guide (especially if we can enforce via linting/CI)

Reply:
#230 would make it easier to use the unicode arrow symbols.